Makefile

```diff
@@ -115,34 +126,18 @@ build: generate fmt vet ## Build manager binary.
 run: manifests generate fmt vet ## Run a controller from your host.
 	go run ./main.go
 
-# If you wish built the manager image targeting other platforms you can use the --platform flag.
-# (i.e. docker build --platform linux/arm64 ). However, you must enable docker buildKit for it.
-# More info: https://docs.docker.com/develop/develop-images/build_enhancements/
-.PHONY: docker-build
-docker-build: test ## Build docker image with the manager.
+
+.PHONY: operator-build
+operator-build: test ## Build operator image with the manager.
 	go mod tidy
-	docker build -t ${IMG} .
-
-.PHONY: docker-push
-docker-push: ## Push docker image with the manager.
-	docker push ${IMG}
-
-# PLATFORMS defines the target platforms for  the manager image be build to provide support to multiple
-# architectures. (i.e. make docker-buildx IMG=myregistry/mypoperator:0.0.1). To use this option you need to:
-# - able to use docker buildx . More info: https://docs.docker.com/build/buildx/
-# - have enable BuildKit, More info: https://docs.docker.com/develop/develop-images/build_enhancements/
-# - be able to push the image for your registry (i.e. if you do not inform a valid value via IMG=<myregistry/image:<tag>> than the export will fail)
-# To properly provided solutions that supports more than one platform you should use this option.
-PLATFORMS ?= linux/arm64,linux/amd64,linux/s390x,linux/ppc64le
-.PHONY: docker-buildx
-docker-buildx: test ## Build and push docker image for the manager for cross-platform support
-	# copy existing Dockerfile and insert --platform=${BUILDPLATFORM} into Dockerfile.cross, and preserve the original Dockerfile
-	sed -e '1 s/\(^FROM\)/FROM --platform=\$$\{BUILDPLATFORM\}/; t' -e ' 1,// s//FROM --platform=\$$\{BUILDPLATFORM\}/' Dockerfile > Dockerfile.cross
-	- docker buildx create --name project-v3-builder
-	docker buildx use project-v3-builder
-	- docker buildx build --push --platform=$(PLATFORMS) --tag ${IMG} -f Dockerfile.cross
-	- docker buildx rm project-v3-builder
-	rm Dockerfile.cross
+	$(CTR_CMD) build -t ${IMG} \
+	--build-arg TARGETOS=$(GOOS) \
+	--build-arg TARGETARCH=$(GOARCH) \
+	--platform=linux/$(GOARCH) .
```

Comment:
Does this help create multi-arch images? I haven't played around with it but my reference would be - https://github.com/prometheus-operator/prometheus-operator/blob/main/scripts/push-docker-image.sh

Reply:
Could you please point to a multi-arch image created using this?

Reply:
> Does this help create multi-arch images? 

Yes, I tried both with podman as well as docker. If do a inspect on the image you will be able to see the architecture which you specified in the platform flag.

> Could you please point to a multi-arch image created using this?

[Kepler](https://github.com/sustainable-computing-io/kepler/blob/f94a6651894f875f556f7494934055dc681ad8ce/Makefile#L179) repo uses this to build multi-arch images

Reply:
I will test this when I have a bit more time, meanwhile, could you please fix the GitHub action as well?

Reply:
Do we want CI to build operator image using docker or podman? Since GH runner supports both it uses podman by default.
Also, we have steps in `integration_test` workflow file and `prepareKubeConfig.sh` which uses docker, and that will require an update accordingly